src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURES_AMOUNT = 5;
+    public static final double MAX_FIGURE_SIDE_VALUE = 100;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        return new Circle(10, Color.WHITE.toString().toLowerCase());
+    }
+
+    public Figure getRandomFigure() {
+        double randomDouble = random.nextDouble(MAX_FIGURE_SIDE_VALUE);
+        String randomColor = colorSupplier.getRandomColor().toString().toLowerCase();
```

Comment:
Use `colorSupplier.getRandomColor().name().toLowerCase()` instead of `colorSupplier.getRandomColor().toString().toLowerCase()` to get the string representation of the enum constant. This aligns with the checklist's guidance to use `name()` for enum constants.